resources/js/components/people/Tasks.vue

```diff
@@ -31,7 +31,7 @@
       </div>
 
       <!-- LIST OF IN PROGRESS TASKS -->
-      <ul>
+      <ul cy-name="tasks-body" :cy-items="inProgress(tasks).map(t => t.id)">
```

Comment:
I don't understand what's this does.

Reply:
this allow cypress to know the id of the items inside the iteration (it unlikely '1').
I just though it's kind a problem to output this in production ...